src/Sulu/Bundle/MediaBundle/Tests/Unit/DependencyInjection/FormatCacheClearerCompilerPassTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Tests\Unit\DependencyInjection;
+
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase;
+use Sulu\Bundle\MediaBundle\DependencyInjection\FormatCacheClearerCompilerPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Test the image command compiler pass
+ */
+class FormatCacheClearerCompilerPassTest extends AbstractCompilerPassTestCase
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function registerCompilerPass(ContainerBuilder $container)
+    {
+        $container->addCompilerPass(new FormatCacheClearerCompilerPass());
+    }
+
+    /**
+     * @test
+     */
+    public function if_compiler_pass_collects_services_by_adding_method_calls_these_will_exist()
```

Comment:
Our convention is to use method names starting with `test` instead of the annotation, and we use camel case for method names. I guess I would even have gone with something like `testCompileCollectsServices` :smile: